tests/py_test.py

```diff
@@ -478,6 +478,39 @@ def test_create_byte_vector(self):
         # 1-byte pad:
         self.assertBuilderEquals(b, [3, 0, 0, 0, 1, 2, 3, 0])
 
+    def test_create_numpy_vector(self):
+        try:
+            imp.find_module('numpy')
+            # if numpy exists, then we should be able to get the
+            # vector as a numpy array
+            import numpy as np
+
+            # Systems endian
+            b = flatbuffers.Builder(0)
+            x = np.array([1, 2, -3], dtype=np.int8)
```

Comment:
Please add tests for some other scalar dtypes (`uint16`, `int64`) and float and double types.

Reply:
done